webapp/src/components/admin_settings/jaas_section.tsx

```diff
@@ -0,0 +1,180 @@
+                                <FormattedMessage
+                                    id='jaas.app-id-description'
+                                    defaultMessage={'Specify your JaaS AppID. You can get the AppID from https://jaas.8x8.vc/#/apikeys .'}
+                                />
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div className='form-group'>
+                    <label className='col-sm-4'>
+                        <FormattedMessage
+                            id='jaas.api-id'
+                            defaultMessage={'Api ID for JaaS Authentication:'}
+                        />
+                    </label>
+                    <div className='col-sm-8'>
+                        <input
+                            className='form-control'
+                            type='input'
+                            maxLength={-1}
+                            onChange={this.props.onApiKeyChange}
+                            value={this.props.apiKey}
+                            disabled={this.props.disabled}
+                        />
+                        <div className='help-text'>
+                            <span>
+                                <FormattedMessage
+                                    id='jaas.api-id-description'
+                                    defaultMessage={'Specify your JaaS Api ID. You can get the Api ID from https://jaas.8x8.vc/#/apikeys .'}
+                                />
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div className='form-group'>
+                    <label className='col-sm-4'>
+                        <FormattedMessage
+                            id='jaas.rsa-key'
+                            defaultMessage={'RSA Private key for JaaS Authentication:'}
+                        />
+                    </label>
+                    <div className='col-sm-8'>
+                        <textarea
+                            className='form-control'
+                            maxLength={-1}
+                            rows={5}
+                            onChange={this.props.onPrivateKeyChange}
+                            value={this.props.privateKey}
+                            disabled={this.props.disabled}
+                        />
+                        <div className='help-text'>
+                            <span>
+                                <FormattedMessage
+                                    id='jaas.rsa-key-description'
+                                    defaultMessage={'Specify your JaaS private key. You can get the private key by generating an API key pair from https://jaas.8x8.vc/#/apikeys OR from https://jaas.8x8.vc/#/start-guide in the 1. Api Key section(NOTE: While generating the key from terminal make sure to set the size to be 2048 and not 4096).'}
```

Comment:
```suggestion
                                    defaultMessage={'Specify your JaaS private key. You can get the private key by generating an API key pair from https://jaas.8x8.vc/#/apikeys OR from https://jaas.8x8.vc/#/start-guide in the "Api Key" section. (NOTE: While generating the key manually make sure to set the size to be 2048 and not 4096).'}
```
This change should also be done in the `i18n/en.json` file.